ivadomed/loader/slice_filter.py

```diff
@@ -4,65 +4,101 @@
 
 
 class SliceFilter(object):
```

Comment:
Just curious, do you think we should rename this to `SlicePatchFilter` now? Or something like `GenericFilter` or `ConfigurableFilter`? 

Also, now that I read through the class a bit, it seems more like a kind of parameterized filter function than a class. 

I have not used many callable class before so pretty cool TIL to see class structured in this way. 

Also I sort of agree with Julien's other commends on the issue, perhaps, SliceSelector or SlicePatchSelector is more 
distinct because in image/3D data domain, filter has many very unique purpose (e.g. gaussian filter) so maybe `selector` would make the purpose of this class more clear.

Reply:
Ok, as discussed in meeting. Let's plan to make the "refactoring" changes separately as it deals with retrocompatiblity with old config files.
Depending on the complexity of the current PR, we may want to do the refactoring before.

Reply:
Yeah, I think it is complicated enough to worry about the refactoring in the future.

Reply:
I agree. Let's keep it simple.